tools/decode-packets/src/main/scala/XToolsV2.scala

```diff
@@ -0,0 +1,157 @@
+import java.io.{BufferedWriter, File, FileWriter}
+import java.nio.charset.CodingErrorAction
+
+import net.psforever.packet.PacketCoding
+import scodec.bits._
+import scodec.Attempt.{Failure, Successful}
+import java.nio.file.{Files, Paths, StandardCopyOption}
+
+import scala.io.{Codec, Source}
+import util.control.Breaks._
+import scala.collection.parallel.CollectionConverters._
+
+object XToolsV2 {
```

Comment:
This and the file should probably match the subproject? No matter what we call it.

Reply:
In light of the discussion below, shall we rename it once we're happy to get rid of the original `xtools`?

Reply:
Sounds good :shipit: